ui.py

```diff
@@ -633,11 +633,27 @@ def draw(self, context):
         mat = context.material
         row = layout.row()
         if mat:
-            row.template_preview(context.material, show_buttons=1)
```

Comment:
So what's the idea with this?  This removes the buttons that allow you to set the object type?